CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## 0.12.10 (Unreleased)
 
+NOTES:
+* The Terraform Plugin SDK is now a standalone Go module: https://github.com/hashicorp/terraform-plugin-sdk. The unofficial plugin SDK in the present repository is now deprecated. Terraform providers should import the new Go module. For more information please see the documentation: https://www.terraform.io/docs/extend/plugin-sdk.html
```

Comment:
> is now deprecated

I think we might want to add some language/warning around "and will not receive updates" or something. And question, will providers who don't migrate get any indication in their development that it is deprecated? (only so we can note in the changelog the concrete (if any) things they will see by _not_ migrating)